bootstrap/schema_ingester.py

```diff
@@ -0,0 +1,198 @@
+'''
+ * ADOBE CONFIDENTIAL
+ * ___________________
+ *
+ *  Copyright 2018 Adobe Systems Incorporated
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Adobe Systems Incorporated and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Adobe Systems Incorporated and its
+ * suppliers and are protected by all applicable intellectual property
+ * laws, including trade secret and copyright laws.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Adobe Systems Incorporated.
+'''
+
+import json
+from helper import setup_logger, http_request
+
+LOGGER = setup_logger(__name__)
+CONTENT_TYPE = "application/json"
+
+
+def get_tenant_id(tenant_id_url, headers):
+    """
+    Get TenantId by making a GET call to "/data/foundation/schemaregistry/stats"
+    :param - url
+    :param - headers
+    :return - tenant id
+
+    """
+    res_text = http_request("get", tenant_id_url, headers)
+    tenant_id = "_" + json.loads(res_text)["tenantId"]
+    LOGGER.debug("tenant_id = %s", tenant_id)
+    return tenant_id
+
+
+def get_class_id(create_class_url, headers, class_title):
+    """
+    Get the classId by making a POST REQUEST to "/data/foundation/schemaregistry/tenant/classes"
+    :param create_class_url:  url
+    :param headers: headers
+    :param class_title: class title
+    :return: class url
+    """
+
+    data_for_create_class = {
+        "type": "object",
+        "title": class_title,
+        "auditable": True,
+        "meta:extends": [
+            "https://ns.adobe.com/xdm/data/time-series"
+        ],
+        "description": class_title,
+        "allOf": [
+            {
+                "$ref": "https://ns.adobe.com/xdm/data/time-series"
+            }
+        ]
+    }
+    headers["content-type"] = CONTENT_TYPE
+    res_text = http_request("POST", create_class_url, headers, json.dumps(data_for_create_class))
+    class_id = json.loads(res_text)["$id"]
+    LOGGER.debug("class_id = %s", class_id)
+    return class_id
+
+
+def get_mixin_id(create_mixin_url, headers, mixin_title, class_id, tenant_id):
+    """
+    Get the mixinId by making a POST REQUEST to "/data/foundation/schemaregistry/tenant/mixins"
+
+    :param create_mixin_url: url
+    :param headers: headers
+    :param mixin_title: mixin title
+    :param class_id: class url
```

Comment:
`class id` ?

Reply:
i like this much better!
Personally I would try to keep YML attributes as close to looking like variables (e.g. Data for creating mixin -> mixins OR mixin_data) but I like the readability of your method.